charts/influxdb/templates/backup-retention-cronjob.yaml

```
@@ -0,0 +1,74 @@
+{{- if .Values.backupRetention.enabled }}
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  name: {{ include "influxdb.fullname" . }}-backup-retention
+  labels:
+    {{- include "influxdb.labels" . | nindent 4 }}
+    app.kubernetes.io/component: backup
+  annotations:
+    {{- toYaml .Values.backupRetention.annotations | nindent 4 }}
+spec:
+  schedule: {{ .Values.backupRetention.schedule | quote }}
+  concurrencyPolicy: Forbid
+  jobTemplate:
```

Should probably also add deadlineseconds here as well.
```suggestion
  startingDeadlineSeconds: {{ .Values. ..
  jobTemplate:
```